new_client/src/components/Home/components/Achievements/Achievements.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect } from "react";
+import Carousel from "react-material-ui-carousel";
+import axios from "axios";
+import Card from "@material-ui/core/Card";
+import { makeStyles } from "@material-ui/core/styles";
+
+import "../Description/Description.scss";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import CardMedia from "@material-ui/core/CardMedia";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+
+const useStyles = makeStyles({
+  carousel: {
+    maxWidth: 700,
+  },
+  root: {
+    maxWidth: 500,
+  },
+  media: {
+    height: 250,
+  },
+  actions: {
+    alignItems: "center",
+    display: "flex",
+  },
+});
+
+const Achievements = (props) => {
+  const [achievements, setAchievements] = useState([]);
+
+  useEffect(() => {
+    axios.get(process.env.REACT_APP_API + "/achievements").then((res) => {
+      setAchievements(res.data["achievements"]);
+    });
+  }, [achievements]);
```

Comment:
We want this useEffect to run only once, so achievements as a dependency is not needed.